src/SampleWebApp/Models/AuthorDataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Data
+{
+    public class AuthorDataContext : DbContext
```

Comment:
The class name is user provided. 
The suggested name that is shown to the users while creating a new Context is <ApplicationName>Context